packages/@okta/i18n/src/properties/login_de.properties

```diff
@@ -778,4 +778,4 @@ oie.success.redirect = Sie werden umgeleitet
 ## Footer
 oie.go.back = Zur\u00FCck
 oie.enroll.switch.authenticator = Melden Sie sich mit einer anderen Methode an
-oie.verification.switch.authenticator = Mit einer anderen Methode verifizieren
\ No newline at end of file
+oie.verification.switch.authenticator = Mit einer anderen Methode verifizieren
```

Comment:
Maybe don't need this change?

Reply:
that's weird, from my end it says these are exactly the same

Reply:
you probably have "Hide whitespace changes" checked. It is in the settings for the PR (gear icon next to "Jump to". 
Or you can check in your url, it will have a url param like `w=1`